.github/workflows/pypi_build.yml

```diff
@@ -0,0 +1,29 @@
+name: Build for PyPi
+
+on:
+  push:
+    tags:
+      - "v*"
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
```

Comment:
in tests we have 3.10, inconsistent or consider just using tox in matrix

Reply:
Oops! I just forgot to change it when I copied it from an old project. Thanks for catching it.